packages/ui-extensions/src/surfaces/admin/globals.ts

```diff
@@ -1,8 +1,10 @@
-import type {ExtensionPoints} from './extension-points';
+import type {ExtensionTargets} from './extension-targets';
 
 export interface ShopifyGlobal {
-  extend<ExtensionPoint extends keyof ExtensionPoints>(
-    extensionPoint: ExtensionPoint,
-    extend: ExtensionPoints[ExtensionPoint],
+  extend<ExtensionTarget extends keyof ExtensionTargets>(
+    extension: {
+      target: ExtensionTarget;
```

Comment:
Sorry, this is my bad, this has to be `extension: ExtensionTarget`

Reply:
Sorry for the confusion, but why would it be `extension: ExtensionTarget`? If anything `extend` should be `extension`,  since `extend` is replaced with `extension` in our JS API. 

Wouldn't this be more appropriate to just be `target: ExtensionTarget`?

Reply:
yes, that's true that it can be `target:ExtensionTarget`. Either way the first param should be a string. Sorry @Fionoble do you mind updating? 

> If anything extend should be extension, since extend is replaced with extension in our JS API.
No, `extension` is the wrapper around `extend`. There's a bunch of stuff it does under the hood. `extend` is the low level API here: https://github.com/Shopify/ui-extensions/blob/a34277d5d1ce5c7fe5fda4d5d388a6db161c6618/packages/ui-extensions/src/utilities/registration.ts#L73

Reply:
Ok, I'm going to follow-suit in my [corresponding PR for checkout-web](https://github.com/Shopify/checkout-web/pull/23530).